### DISCUSSION TITLE
[Idea] Automatically add a semicolon after } for struct and class declarations

### DISCUSSION THREAD

**Discussion by sryze** [microsoft/vscode-cpptools#10640] [Ideas]
Feature idea: When you type `class ClassName {` VS Code automatically adds a closing `}`, but it would be extra nice if it added the `;` as well, as required by C++ syntax, so that you don't have to go and add it manually.

**Comment by sean-mcmanus** [comment]
This is already being tracked by https://github.com/microsoft/vscode-cpptools/issues/3087 , but the root issue is with VS Code (not our extension), but it didn't get enough votes and was closed: https://github.com/microsoft/vscode/issues/67090
